content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -88,66 +102,114 @@ Events:
   1m           1m          1        {replicaset-controller }             Normal      SuccessfulCreate  Created pod: frontend-qhloh
   1m           1m          1        {replicaset-controller }             Normal      SuccessfulCreate  Created pod: frontend-dnjpy
   1m           1m          1        {replicaset-controller }             Normal      SuccessfulCreate  Created pod: frontend-9si5l
-$ kubectl get pods
+```
+
+And lastly checking on the pods brought up:
+```shell
+kubectl get pods
+```
+
+Will yield pod information similar to
+```shell
 NAME             READY     STATUS    RESTARTS   AGE
 frontend-9si5l   1/1       Running   0          1m
 frontend-dnjpy   1/1       Running   0          1m
 frontend-qhloh   1/1       Running   0          1m
 ```
 
-## Writing a ReplicaSet Spec
+## Non-Template Acquisitions
 
-As with all other Kubernetes API objects, a ReplicaSet needs the `apiVersion`, `kind`, and `metadata` fields.  For
-general information about working with manifests, see [object management using kubectl](/docs/concepts/overview/object-management-kubectl/overview/).
+A ReplicaSet is not limited to owning pods specified by its template. Take the previous frontend ReplicaSet example,
```

Comment:
Before you show the example of non-template acquisitions, I think you should make a statement that recommends against manually creating Pods that match some ReplicaSet's selector. Then the example will serve as a "here's why".